packages/editor/src/bindings/post-meta.js

```diff
@@ -3,15 +3,15 @@
  */
 import { useEntityProp } from '@wordpress/core-data';
 import { useSelect } from '@wordpress/data';
-import { __ } from '@wordpress/i18n';
+import { _x } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
 import { store as editorStore } from '../store';
 
 export default {
 	name: 'core/post-meta',
-	label: __( 'Post Meta' ),
+	label: _x( 'Post Meta' ),
```

Comment:
We should mirror: https://github.com/WordPress/wordpress-develop/blob/acad00ecabcf44a6c4e552c4fd518ee708fd131f/src/wp-includes/block-bindings/post-meta.php#L52

Reply:
You're right! Done in [this commit](https://github.com/WordPress/gutenberg/pull/58787/commits/653a477f06cf7456002dcd50df5a36c92c55d9df).